### PR TITLE
Using /var/flexlm/adsk_server.lic and docs update

### PR DESCRIPTION
**Pull request by haysclark**
- Changing the default license path to ‘/var/flexlm/adsk_server.lic’ which aligns with Autodesk Help Desk articles.
- Updating documentation which did not call out the expected license paths. Props to @hyperknot
- Upgrading example to read-only file to file mapping which additionally calls out the expected license path. Props to @hyperknot

Resolves: #1

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+- Upgrading example to read-only file to file mapping which additionally calls out the expected license path. Props to @hyperknot
+
+### BREAKING CHANGES
+- Changing the default license path to __/var/flexlm/adsk_server.lic__ which aligns with Autodesk Help Desk articles.
+
+<a name="0.9.0"></a>
+# 0.9.0 (2016-10-10)
+
+### Initial Release
+- A Docker container for running _lmstat_.
```
Comment (hyperknot) [inline CHANGELOG.md]: maybe lmgrd?

**Comment by haysclark** [comment]
Interesting, I didn't know that.  I'm hunting for some docs on the -l flag.  I'm guessing that if not included it will output to stdout/stderr?  Good catch!

**Comment by hyperknot** [comment]
Yes, that's how stdout behaves with Docker! If either `-l ""` or `-l -` or something similar would work then I guess we could use ARGS to enable/disable logging.

Actually, with this changelog, your filesystem based log file is not going to be visible to the host.

**Comment by haysclark** [comment]
@hyperknot I totally see your point. Logging to _/var/log/flexlm/lmgrd.log_ is lame compared to just logging it to Docker and utilizing ```docker log```.  I'll update this default behavior and post the fix in a bit.

**Comment by haysclark** [comment]
@hyperknot Less is more, no default arguments. 😄  Hopefully, this will result is less confusion, better logging, etc.  The logging change will also be helpful when utilizing other utilities packaged with the FlexLM.

Sorry about the confusion about logging, I read what you mentioned in your Issue #1.  However, I did not make the connection that _lmgrd_ output everything to stdout if the _-l_ flag is not provided.

**Comment by hyperknot** [comment]
Thanks a lot for all these changes. These were minor things for me, but I believe for the future users they might save quite a lot of time of figuring everything out!

**Pull request merged**